shell/packages/sandstorm-accounts-ui/login_buttons.js

```diff
@@ -2,6 +2,9 @@
 var loginButtonsSession = Accounts._loginButtonsSession;
 
 var helpers = {
+  isCurrentRoute: function (routeName) {
+    return Router.current().route.getName() == routeName;
```

Comment:
This introduces a reference to a deprecated global into a package, which we were trying not to do...